### ISSUE TITLE
Application does not load after updating Prism.Plugin.Popups

### ISSUE THREAD

**Issue by GyllionElderen** [dansiegel/Prism.Plugin.Popups#67]
I found out that after updating Prism.Plugin.Popups to version 2, it is conflicting with INavigationService. See the following image below:

![image](https://user-images.githubusercontent.com/32705616/39422339-37fcc4a4-4c6d-11e8-95d3-5be63e77dcea.png)

I have already set up the Rg.Plugins.Popup correctly. It was running without the Prism.Plugin.Popups package installed.

These are the packages I'm using:
- Xamarin.Forms: 2.5.1.444934
- Rg.Plugins.Popup: 1.1.4.154-pre
- Prism.Plugin.Popups: 2.0.0-pre6

**Comment by dansiegel** [comment]
you shouldn't have an exception thrown by simply not registering the PopupNavigationService as it would simply push your PopupPage's the same as it would a ContentPage. Is your issue resolved? If it isn't, please provide a repo app so I can better assist you.

**Comment by GyllionElderen** [comment]
@dansiegel Thanks the reply. The problem has been resolved, but I found something interesting.

I made another project to see if I can actually reproduce this exception. After making a brand new project and installing the Rg.Plugins.Popup and Prism.Plugin.Popups packages, everything went fine, even without registering the PopupNavigationService. When updating Xamarin.Forms to version 3.0.0.427558-pre4, the exception occured again. Here's proof:

![error](https://user-images.githubusercontent.com/32705616/39464426-85fb9cd8-4d1d-11e8-8044-a6f523c9493a.PNG)

In here, the PopupNavigationService is already registered. In my last project, I upgraded Xamarin.Forms to 3.0.0.427558-pre4 and then downgraded to 2.5.1.444934. There was probably something stuck in the project or I did not rebuild properly.
